sqlglot/diff.py

```diff
@@ -109,16 +115,41 @@ def __init__(self, f: float = 0.6, t: float = 0.6) -> None:
         self.t = t
         self._sql_generator = Dialect().generator()
 
-    def diff(self, source: exp.Expression, target: exp.Expression) -> t.List[Edit]:
+    def diff(
+        self,
+        source: exp.Expression,
+        target: exp.Expression,
+        matchings: t.List[t.Tuple[exp.Expression, exp.Expression]] | None = None,
```

Comment:
matches

Reply:
I think matching is a valid term in this context (e.g. https://en.wikipedia.org/wiki/Matching_(graph_theory)).

Reply:
sounds werid to me

Reply:
Yeah, I prefer matching over matches. "Matches" sound like something you use to start a fire.